injector/grpc_disruption.go

```diff
@@ -81,10 +89,12 @@ func (i *GRPCDisruptionInjector) Clean() error {
 		return nil
 	}
 
-	conn, err := connectToServer(i.serverAddr)
+	conn, err := i.connectToServer()
 	if err != nil {
 		i.config.Log.Errorf(err.Error())
-		return err
+		// swallow connection err because there is nothing else we can do with the error
```

Comment:
But if we are failing to connect, then we can't have cleaned, so why do we want to allow `clean` to exit successfully?

Reply:
my reason is that there is nothing this command can do when server is unreachable